packages/ui/src/components/Notification/index.tsx

```diff
@@ -47,6 +47,7 @@ const styles = {
     ${PREFIX}__toast-body {
       margin: 0;
       padding: 0;
+      display: none;
```

Comment:
Why do we need this?

Reply:
I saw an empty vertical space when using the notification